src/web-animations-next-animation.js

```diff
@@ -31,8 +34,20 @@
   // TODO: add an effect getter/setter
   scope.Animation.prototype = {
     _rebuildUnderlyingAnimation: function() {
-      if (this._animation) {
+      var oldPlaybackRate;
+      var oldPaused;
+      var oldStartTime;
+      var oldCurrentTime;
+      var oldHoldTime;
+      var hasUnderlying = this._animation ? true : false;
+      if (hasUnderlying) {
+        oldPlaybackRate = this.playbackRate;
+        oldPaused = this._paused;
+        oldStartTime = this.startTime;
+        oldCurrentTime = this.currentTime;
+        oldHoldTime = this._holdTime;
         this._animation.cancel();
+        this._animation._wrapper = this._animation;
```

Comment:
> Where is _wrapper used?

It is used in the ticker in newUnderlyingAnimationForGroup, in group-constructors.js. Pointing this animation's wrapper at itself was a bit of a hack. I should probably make it null and then check for null in the ticker.